testnets/atomonetestnet1/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "atomonetestnet1",
+  "assets": [
+    {
+      "description": "The native staking and governance token of AtomOne",
+      "denom_units": [
+        {
+          "denom": "uatone",
+          "exponent": 0
+        },
+        {
+          "denom": "atone",
+          "exponent": 6
+        }
+      ],
+      "base": "uatone",
+      "name": "Atone",
+      "display": "atone",
+      "symbol": "ATONE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/atomonetestnet1/images/atomone.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/atomonetestnet1/images/atomone.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/atomonetestnet1/images/atomone.png",
```

Comment:
Please remove the uploaded images here under testnets/atomonetestnet1/images/ since they are duplicates of mainnet logo.